pyccl/halos/profiles/gaussian.py

```diff
@@ -26,7 +26,7 @@ class HaloProfileGaussian(HaloProfile):
 
     @deprecated()
     @warn_api
-    def __init__(self, *, r_scale, rho0, mass_def):
+    def __init__(self, *, r_scale, rho0, mass_def=None):
```

Comment:
So the point is, for v3 we need to remove the `=None` bit?

Reply:
yes, we are making it a required argument